python/nvstrings/nvtext.py

```diff
@@ -346,3 +346,27 @@ def scatter_count(strs, counts):
     if rtn is not None:
         rtn = nvs.nvstrings(rtn)
     return rtn
+
+
+def porter_stemmer_measure(strs, devptr=0):
+    """
+    Calculates porter stemmer measure of each string
+
+    Parameters
+    ----------
+    strs : nvstrings
+        The strings for this operation
+    devptr : GPU memory pointer
+        Must be able to hold at least strs.size() of int32 values.
+
+    Examples
+    --------
+    >>> import nvstrings, nvtext
+    >>> s = nvstrings.to_device(["hello","goodbye",""])
+    >>> n = nvtext.porter_stemmer_measure(s)
+    >>> print(n)
```

Comment:
Show the output of the `print(n)` in here as well.

Reply:
Added, 

https://github.com/rapidsai/cudf/blob/170e59be2442c079e56a950799c4c5e954712f33/python/nvstrings/nvtext.py#L377